internal/resources/snapshot_backup_schedule_schema.go

```diff
@@ -0,0 +1,22 @@
+package resources
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+func SnapshotBackupScheduleSchema() schema.Schema {
+	return schema.Schema{
+		MarkdownDescription: "Manages snapshot backup schedule resource",
+		Attributes: map[string]schema.Attribute{
+			"organization_id": WithDescription(stringAttribute([]string{required, requiresReplace}, validator.String(stringvalidator.LengthAtLeast(1))), "The GUID4 ID of the organization."),
+			"project_id":      WithDescription(stringAttribute([]string{required, requiresReplace}, validator.String(stringvalidator.LengthAtLeast(1))), "The GUID4 ID of the project."),
+			"id":              WithDescription(stringAttribute([]string{required, requiresReplace}, validator.String(stringvalidator.LengthAtLeast(1))), "The GUID4 ID of the cluster."),
+			"interval":        WithDescription(int64Attribute(optional, computed), "The interval of the snapshot backup schedule."),
+			"retention":       WithDescription(int64Attribute(optional, computed), "The retention of the snapshot backup schedule."),
+			"start_time":      WithDescription(stringAttribute([]string{optional, computed}), "The start time of the snapshot backup schedule."),
+			"copy_to_regions": WithDescription(stringSetAttribute(optional, computed), "The regions to copy the snapshot backup to."),
```

Comment:
should all these attributes be `computed` ?  

`computed` means if the user does not set an option, the backend will set some value.  is this the case ?

to check this create a backup schedule with curl using only required attributes, which means there is no request body:

```
curl --request PUT \
       --url {host}/v4/organizations/{orgId}/projects/{projectId}/clusters/{clusterId}/cloudsnapshotbackupschedule
       --header 'Authorization: Bearer {token}'
```

then do GET and see what values those attributes have in the response body.